src/pages/contact_information.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Card,
+  Image,
+  Text,
+  Button,
+  Group,
+  CardSection,
+} from "@mantine/core";
+
+interface ContactCardProps {
+  title: string;
+  operationTitle: string;
+  operationTime: string;
+  imgUrl: string;
+  email: string;
+}
+
+const imageUrl = "/assets/images/logos/registrar.png";
+
+const contactCards = [
+	{
+		imgUrl: imageUrl,
+		title: "Academic Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "ug_academic@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Registra",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "registrar.bkk@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Library",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "library@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Finance",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "cashier_rm9@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "IT Support",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "support@stamford.edu",
+	},
+	{
+		imgUrl: imageUrl,
+		title: "Student Service",
+		operationTitle: "Operation Hours:",
+		operationTime: "Mon-Fri 08:30 - 17.30",
+		email: "international-service@stamford.edu",
+	},
+];
+
+const ContactCard = ({
```

Comment:
maybe later we should extract this to be in a separate file?

Reply:
yeaa, we can. It kinda looks bad here